src/gausskernel/storage/access/k2/pg_gate_api.cpp

```diff
@@ -371,12 +371,16 @@ K2PgStatus PgGate_NewDropTable(K2PgOid database_oid,
                             bool if_exist,
                             K2PgStatement *handle) {
   elog(LOG, "PgGateAPI: PgGate_NewDropTable %d, %d", database_oid, table_oid);
-  return K2PgStatus::NotSupported;
+  K2PgStatus status = pg_gate->GetCatalogClient()->DeleteTable(database_oid, table_oid);
```

Comment:
The actual DropTable should be executed in PgGate_ExecDropTable(). Could you combine the following two methods,

K2PgStatus PgGate_NewDropTable(K2PgOid database_oid,
                            K2PgOid table_oid,
                            bool if_exist,
                            K2PgStatement *handle)
                            
and 

K2PgStatus PgGate_ExecDropTable(K2PgStatement handle)

into one method

K2PgStatus PgGate_ExecDropTable(K2PgOid database_oid,
                            K2PgOid table_oid,
                            bool if_exist)

Reply:
Done